### ISSUE TITLE
fine-tuning

### ISSUE THREAD

**Issue by my-221** [fc2869/lo-fit#4]
May I ask, what was the data volume during your fine-tuning process? How many training epochs did you conduct? Is it necessary for the loss to converge?

**Comment by fc2869** [comment]
Hi! Thank you for your interest in our work! 

We use the following dataset composition: (1) TruthfulQA: train/dev/test = 326/82/407; (2) CLUTRR: train/dev/test = 300/450/450; (3) MQuAKE: train/dev/test = 134/95/864.

For all tasks, we train for 5 epochs. Note that it is not necessary for the loss to converge because of the small data size.

You can also find the details of the hyperparameters in Appendix D of our paper (https://arxiv.org/pdf/2406.01563) as well as in the train scripts to reproduce the results. 

Let me know if you have any other question and we are more than happy to help!
